easybuild/easyconfigs/r/ROOT/ROOT-6.10.08-foss-2017b-Python-2.7.14.eb

```diff
@@ -38,6 +41,9 @@ dependencies = [
 configopts = '-Dxrootd=OFF -Dmysql=OFF -Dkrb5=OFF -Dodbc=OFF '
 configopts += '-Doracle=OFF -Dpgsql=OFF -Dqt=OFF '
 configopts += '-Dunuran=ON -Dtable=ON -Dexplicitlink=ON -Dminuit2=ON '
-configopts += '-Droofit=ON'
+configopts += '-Droofit=ON '
+
+# Needed if CFITSIO was build with cURL (optional dependency)
```

Comment:
@casparvl The comment in https://github.com/easybuilders/easybuild-easyconfigs/pull/7347/files#diff-3f9708d03f22d0e104c6c7dc0017d7e2R51 was actually more descriptive, please use it heretoo?